src/Handler.php

```diff
@@ -12,24 +12,85 @@ class Handler
     protected $client;
 
     /**
+     * The previously registered error handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousErrorHandler;
+
+    /**
+     * The previously registered exception handler as returned by the interpreter.
+     *
+     * @var callable|null
+     */
+    protected $previousExceptionHandler;
+
+    /**
      * Register our exception handler.
      *
      * @param \Bugsnag\Client|string|null $client client instance or api key
+     * @param bool $callPrevious whether or not to call the previous handlers
      *
      * @return static
      */
-    public static function register($client = null)
+    public static function register($client = null, $callPrevious = true)
```

Comment:
We should be very careful adding new parameters to public or protected methods in PHP since it's technically a BC break. A way around this would be to define a new method called `registerWithPrevious`.